src/main/java/com/purbon/kafka/topology/model/impl/TopicImpl.java

```diff
@@ -20,36 +20,29 @@
 public class TopicImpl implements Topic, Cloneable {
 
   private static final String DEFAULT_TOPIC_NAME = "default";
+  private static final String DEFAULT_PARTITION_COUNT = "3";
+  private static final String DEFAULT_REPLICATION_FACTOR = "2";
 
-  @JsonInclude(Include.NON_EMPTY)
-  private Optional<String> dataType;
-
-  @JsonInclude(Include.NON_EMPTY)
-  private List<TopicSchemas> schemas;
-
-  private String name;
-
-  private List<Producer> producers;
-  private List<Consumer> consumers;
-
-  @JsonInclude(Include.NON_EMPTY)
-  private String plan;
-
-  private Map<String, String> metadata;
-
-  private Map<String, String> config;
-  @JsonIgnore private TopologyBuilderConfig appConfig;
+  @JsonIgnore private TopologyBuilderConfig builderConfig;
   @JsonIgnore private Map<String, Object> context;
+  @JsonIgnore private String projectPrefix;
 
+  private String name;
   private int partitionCount;
   private short replicationFactor;
+  private Map<String, String> config;
 
-  @JsonIgnore private String projectPrefix;
-  private static String DEFAULT_PARTITION_COUNT = "3";
-  private static String DEFAULT_REPLICATION_FACTOR = "2";
+  private Map<String, String> metadata;
+
+  private List<Producer> producers;
```

Comment:
why the reordering of many of these attributes for the class?

Reply:
was hard to figure out which fields were ignored or not. Just to make that easier to read.